master/getting-started/bare-metal/bare-metal.md

```diff
@@ -388,11 +388,14 @@ policy, {{site.prodname}} has a failsafe mechanism that keeps various pinholes o
 in the firewall.
 
 By default, {{site.prodname}} keeps port 22 inbound open on *all* host endpoints,
-which allows access to ssh; as well as inbound and outbound communications to
-ports 2379, 2380 (etcd's default ports), and to ports 6666, 6667 (etcd's ports
+which allows access to ssh; outbound UDP port 53 for DNS queries, 
+inbound UDP port 68 and outbound UDP 67 for DHCP; as well as inbound and outbound 
```

Comment:
this might be easier to digest in table format now that we have more than a couple of ports

Reply:
I think that's a great idea. I'll update it